### PR TITLE
[documentation] Describe CMake usage. JB#48361

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: doc/src/sdk-user-manual-cmake.qdoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**
+** Copyright (C) 2020 Open Mobile Platform LLC.
+** Contact: Ville Nummela <ville.nummela@jolla.com>
+** See README and LICENSE.txt for licence details.
+**
+****************************************************************************************/
+
+/*!
+\page sdk-user-manual-cmake..html
+\previouspage {{Building system packages}
```
Comment (martyone) [inline doc/src/sdk-user-manual-cmake.qdoc]: double {{

File: doc/src/sdk-user-manual-cmake.qdoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+It is possible to create a CMake based Sailfish OS application project either 
+
+\list
+    \li from Qt Creator, by using the "Sailfish OS Qt Quick Application" wizard as 
+        usually, but selecting "CMake" on the "Build System" page, or
+    \li with "sfdk init" command by passing the -b|--builder option with "cmake" as 
+        argument.
+\endlist
+Please check the generated project for hints how to use CMake with Sailfish SDK.
```
Comment (martyone) [inline doc/src/sdk-user-manual-cmake.qdoc:28]: Is it desired that this is not separated with blank line?

**Review by martyone — APPROVED** [review]
otherwise LGTM

File: doc/src/sdk-user-manual-cmake.qdoc (hunk 1 above)
Comment (vigejolla) [reply doc/src/sdk-user-manual-cmake.qdoc]: fixed

File: doc/src/sdk-user-manual-cmake.qdoc (hunk 2 above)
Comment (vigejolla) [reply doc/src/sdk-user-manual-cmake.qdoc:28]: added blank line

**Pull request merged**